src/applications/appeals/995/components/MissingInfo.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import recordEvent from 'platform/monitoring/record-event';
+
+const MissingInfo = ({ hasSsn, hasDob }) => {
+  const hasSsnOrDob = [
+    hasSsn ? '' : 'Social Security number',
+    hasDob ? '' : 'date of birth',
+  ].filter(Boolean);
+  const missing = hasSsnOrDob.join(' and ');
```

Comment:
If DOB and SSN do exist will `missing` still join the empty strings?

Reply:
The `join` won't work with an array with 1 or less entry. So we will end up with an empty string